frontend/src/Components/IndicatorBars/PlantDetailBar.js

```diff
@@ -1,28 +1,68 @@
 import PropTypes from 'prop-types'
+import { useState } from 'react'
 import styled from 'styled-components'
+import { SettingsIcon } from '../Icons'
+import SettingCareIntervalModal from '../Modals/SettingCareIntervalModal'
 
 export default function PlantDetailBar({ daysLeft, weeksLeft }) {
   PlantDetailBar.propTypes = {
     daysLeft: PropTypes.number.isRequired,
     weeksLeft: PropTypes.number.isRequired,
   }
 
+  const [isCareSettingShown, setIsCareSettingShown] = useState()
```

Comment:
What do you think about just `showCareSetting` for a simpler name?

Maybe init to empty object, or better 
```
{
    water: false,
    fertilizer: false
}
```
to make intention clearer. 

Then later simply `showCareSetting.water &&`

Can both modals be open at the same time, if not, make sure that state is never reached.

Reply:
Good point, at the moment it should never be reached, but I'm going to change it with the init state..

Reply:
I think you can use the function with no modification, say you click the fertilizer button, your state becomes 
```
{
    fertilizer: true
}
```
and `showCareSetting.water` will be falsy and vice versa for the watering button. Prevents the state of both being true, as well.